### PR TITLE
Changing the XML reference formatting for required and platforms attr…

### PR DESCRIPTION
**Pull request by rakatyal**
…ibutes

### CODE REVIEW

File: www/static/css-src/main.scss [hunk 1]
```diff
@@ -173,6 +173,12 @@ code.prettyprint {
     border: none !important;
 }
 
+mark {
+    background-color: lightblue;
+    padding: 0.1px 0.2px 0.1px 0.2px;
+    margin-right: 7px;
+}
+
```
Comment (riknoll) [inline www/static/css-src/main.scss]: This should probably go in `_docs.scss`

Comment (rakatyal) [inline www/static/css-src/main.scss]: Right.

**Comment by nikhilkh** [comment]
LGTM

**Comment by riknoll** [comment]
LGTM

**Pull request closed without merging**